loadtest/control/simulcontroller/controller.go

```diff
@@ -35,7 +36,7 @@ func New(id int, user user.User, config *Config, status chan<- control.UserStatu
 		return nil, errors.New("nil params passed")
 	}
 
-	if err := config.IsValid(); err != nil {
+	if err := defaults.Validate(*config); err != nil {
```

Comment:
Does it make sense to stop passing configs as pointers now that we have our validation rules or is there a case where it's still needed?
Ideally I'd like to move away from pointers to config objects unless they are strictly necessary (modification of the same object is needed, e.g. `defaults.Set()`).

Reply:
The pointers are not needed actually if no modification is made. Will update accordingly 👍